app/main.py

```diff
@@ -1 +1,65 @@
-# write your code here
```

Comment:
Isn't it necessary to create skills as a class attribute in the task conditions? I can't pass the last test when skills is an instance attributes